azure-functions/src/durable/orchestrator_output.rs

```diff
@@ -0,0 +1,56 @@
+use crate::rpc::{typed_data::Data, TypedData};
+use serde_json::Value;
+
+/// # Orchestrator Output
+/// 
+/// Type returned by Orchetrator Functions for Durable Functions
+
+struct OrchestratorOutput(serde_json::Value);
+
+impl<T> From<T> for OrchestratorOutput
+where 
+    T: Into<Value>,
+{
+    fn from(t: T) -> Self {
+        OrchestratorOutput(t.into())
+    }
+}
+
+#[doc(hidden)]
+impl Into<TypedData> for OrchestratorOutput {
```

Comment:
Unlike `ActivityOutput`, this should just implement `Into<Value>` and return `self.0`.

`ActivityOutput` is returned in the RPC data.  For the `OrchestrationOutput`, it would be in the execution result (which stores an `Value`).